seqr/utils/elasticsearch/es_search.py

```diff
@@ -824,6 +825,20 @@ def _parse_genome_versions(self, result, index_name, hit):
             'liftedOverPos': lifted_over_pos,
         })
 
+    def _parse_transloc_xstop(self, result, index_name):
+        xstop = result.pop(XSTOP_FIELD, None)
+        if self.index_metadata[index_name]['genomeVersion'] != GENOME_VERSION_GRCh38 or\
+                self.index_metadata[index_name]['datasetType'] != Sample.DATASET_TYPE_SV_CALLS or\
+                self.index_metadata[index_name]['sampleType'] != Sample.SAMPLE_TYPE_WGS:
+            return
+        if xstop:
+            endChrom, end = get_chrom_pos(xstop)
+            if endChrom != result['chrom'] and result['svType'] == TRANSLOCATION_TYPE:
```

Comment:
Maybe we should remove the condition of `endChrom != result['chrom']` because the translocation can happen in the same chromosome. We may still need to display the start and the target coordination separately when the chromosomes are the same.

Reply:
Some of the CPX and INS variants have a `CHR2` which is different from the `locus.contig`. I don't think we should display the second coordination for these variants. Changing the `end` fields with the `END2` values could be an error for these variants. The followings are some examples of these CPX and INS variants. 
![image](https://user-images.githubusercontent.com/33550059/159768460-fc3c9a19-cbf7-4649-9d57-c268dc953d85.png)

Reply:
I'm happy to have the UI code check if its a translocation, and not display endChrom otherwise. I'm not okay dong this server side. My initial comment here still stands

Reply:
for all of these examples it looks like end is the same as start, and end2 is different so I think we absolutely want these to display the same way as translocations

Reply:
My concern for these examples is that the CHR2/END2 don't conform to the definition. The END2 is for the CHR2 per the definition. But If you check the CPX_INTERVALS in the above examples, you can see the END2 values don't look like for CHR2. In the contrast, they look more like they are for the locus.contig. I'm also not sure about the CHR2/END2 for the INS variants. That's why I wanted to exclude the non-translocation types when generating the `endChrom`.

Reply:
Change the logic so "end" is always coming from xstop, as I asked above. Change the UI logic to appropriately display translocations and complex variants. For these complex variants it looks like we don't want to show either END or END2, given that END is the same value as start. Therefore, the following UI will work and the server side code here will be clean and maintainable as I suggested. 
```
<b>
  <UcscBrowserLink variant={variant} includeEnd={!!svType && !endChrom} />
  {endCrom && svType === TRANSLOCATION_SV && 
    <span>
      ;&nbsp;
      <UcscBrowserLink variant={variant} isEnd />
    </span>
  }
</b>
```

Reply:
Sounds good.

Reply:
We always display the second coordination for the translocation SVs even if the CHR2 is the same as the locus.contig. Is that right?

Reply:
Yes that is correct. So instead of `endCrom && svType === TRANSLOCATION_SV` the conditional for showing the end coordinate as a separate link should just be svType === TRANSLOCATION_SV` . Good catch